simpleclient/src/main/java/io/prometheus/client/Reservoir.java

```diff
@@ -0,0 +1,85 @@
+package io.prometheus.client;
+
+import java.util.Arrays;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Created by warebot on 12/21/15.
```

Comment:
This comment doesn't belong here, the AUTHORs file and source control history cover this.